whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/attrs/Inet6numStatus.java

```diff
@@ -99,4 +99,8 @@ public boolean isValidOrgType(final OrgType orgType) {
     public String toString() {
         return literalStatus.toString();
     }
+
+    public boolean isAssignment(){
+        return this.equals(ASSIGNED) || this.equals(ASSIGNED_ANYCAST) || this.equals(ASSIGNED_PI);
+    }
```

Comment:
There is a attribute in ManagedAttributeSearch class : INET6NUM_ASSIGNMENT_STATUSES. We are not including  status ASSIGNED there but here we are which one is correct?